ethane-abi/src/parameter/mod.rs

```diff
@@ -213,9 +213,9 @@ mod test {
         index += bytes_read;
 
         assert_eq!(index, 128);
-        assert_eq!(addr.to_string(), String::from("0xffffffffffffffffffffffffffffffffffffffff"));
-        assert_eq!(a.to_string(), String::from("2815"));
-        assert_eq!(b.to_string(), String::from("3071"));
-        assert_eq!(c.to_string(), String::from("3327"));
+        assert_eq!(addr.to_string(), String::from("0x000000000000000000000000ffffffffffffffffffffffffffffffffffffffff"));
+        assert_eq!(a.to_string(), String::from("0xaff"));
```

Comment:
Can you comment the decimal number, just to know it.

Reply:
Sure, I added the decimal equivalents to lines 217-219. The address on 216 is too large however, but it should just match the raw data on line 199.